From c6399409a8aac25081a7218823e369cc4f82de44 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=8D=C3=B1igo=20Huguet?= <ihuguet@redhat.com>
Date: Wed, 24 May 2023 15:00:38 +0200
Subject: [PATCH 121/142] wifi: rtw89: coex: Update Wi-Fi Bluetooth coexistence
 version to 7.0.0
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Bugzilla: https://bugzilla.redhat.com/2207499

commit de06588cb95cb3bbece81e9838547c1a3c2aa023
Author: Ching-Te Ku <ku920601@realtek.com>
Date:   Tue Jan 17 19:41:09 2023 +0800

    wifi: rtw89: coex: Update Wi-Fi Bluetooth coexistence version to 7.0.0

    Should update the driver with the supported firmware version of the
    below item.
    Bluetooth firmware BT_Coex_Ver: 0x07
    Wi-Fi firmware version: RTL8852C->v0.27.56.10, RTL8852A->v0.24.36

    Signed-off-by: Ching-Te Ku <ku920601@realtek.com>
    Signed-off-by: Ping-Ke Shih <pkshih@realtek.com>
    Signed-off-by: Kalle Valo <kvalo@kernel.org>
    Link: https://lore.kernel.org/r/20230117114109.4298-9-pkshih@realtek.com

Signed-off-by: Íñigo Huguet <ihuguet@redhat.com>
---
 drivers/net/wireless/realtek/rtw89/coex.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/realtek/rtw89/coex.c b/drivers/net/wireless/realtek/rtw89/coex.c
index 89214f2db9851..bcf483cafd203 100644
--- a/drivers/net/wireless/realtek/rtw89/coex.c
+++ b/drivers/net/wireless/realtek/rtw89/coex.c
@@ -9,7 +9,7 @@
 #include "ps.h"
 #include "reg.h"
 
-#define RTW89_COEX_VERSION 0x06030013
+#define RTW89_COEX_VERSION 0x07000013
 #define FCXDEF_STEP 50 /* MUST <= FCXMAX_STEP and match with wl fw*/
 
 enum btc_fbtc_tdma_template {
-- 
2.13.6

